site/networking.xml

```diff
@@ -98,6 +104,13 @@ limitations under the License.
           to listen on localhost only for both IPv4 and IPv6:
 
 <pre class="code">
+listeners.tcp.1 = 127.0.0.1:5672
+listeners.tcp.2 = ::1:5672
```

Comment:
Is it possible to indicate an IP address without the port (so the default port is used)?

If yes, we must enclose IPv6 addresses inside square brackets (`[::1]`) like we do in URLs.